cmake/Tools/Platform/Android/android_post_build.py

```diff
@@ -0,0 +1,256 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+import argparse
+import re
+import shutil
+import sys
+
+import platform
+import logging
+
+from packaging.version import Version
+from pathlib import Path
+
+logging.basicConfig(format='%(levelname)s: %(message)s', level=logging.INFO)
+logger = logging.getLogger('o3de.android')
+
+ANDROID_ARCH = 'arm64-v8a'
+
+ASSET_MODE_PAK     = 'PAK'
+ASSET_MODE_LOOSE   = 'LOOSE'
+SUPPORTED_ASSET_MODES = [ASSET_MODE_PAK, ASSET_MODE_LOOSE]
+ASSET_PLATFORM_KEY = 'android'
+
+SUPPORTED_BUILD_CONFIGS = ['debug', 'profile', 'release']
+
+MINIMUM_ANDROID_GRADLE_PLUGIN_VER = Version("4.2")
+
+IS_PLATFORM_WINDOWS = platform.system() == 'Windows'
+
+PAK_FILE_INSTRUCTIONS = "Make sure to create release bundles (Pak files) before building and deploying to an android device. Refer to " \
+                        "https://www.docs.o3de.org/docs/user-guide/packaging/asset-bundler/bundle-assets-for-release/ for more" \
+                        "information."
+
+
+class AndroidPostBuildError(Exception):
+    pass
+
+
+def copy_or_create_link(src: Path, tgt: Path):
+    """
+    Copy or create a link/junction depending on the source type. If this is a file, then perform file copy from the
+    source to the target. If it is a directory, then create a junction(windows) or symlink(linux) from the source to
+    the target
+
+    :param src:             The source to copy/link from
+    :param tgt:             The target tp copy/link to
+    """
```

Comment:
does this really copy anymore? it appears to just make links.

Reply:
Fixed